asyn/drvAsynSerial/drvAsynSerialPort.c

```diff
@@ -526,14 +531,23 @@ setOption(void *drvPvt, asynUser *pasynUser, const char *key, const char *val)
         }
 #endif
     }
-    else if (epicsStrCaseCmp(key, "break") == 0) {
-        unsigned break_len;
-        if(sscanf(val, "%u", &break_len) != 1) {
+    else if (epicsStrCaseCmp(key, "break_duration") == 0) {
+        unsigned break_duration;
+        if(sscanf(val, "%u", &break_duration) != 1) {
+            epicsSnprintf(pasynUser->errorMessage, pasynUser->errorMessageSize,
+                                                                "Bad number");
+            return asynError;
+        }
+        tty->break_duration = break_duration;
+    }
+    else if (epicsStrCaseCmp(key, "break_delay") == 0) {
+        unsigned break_delay;
+        if(sscanf(val, "%u", &break_delay) != 1) {
```

Comment:
A lot of other bits of code here are protected by a `#ifdef vxWorks` construct, do I need to do the same?